src/core/Table/hooks/useColumnVisibility.tsx

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { ColumnInstance, HeaderProps, Hooks } from 'react-table';
+import { Checkbox } from '../..';
```

Comment:
Wondering, if this path would not break any builds...
I do remember we had something with it.. @bentleyvk maybe you recall?

Reply:
I think there were problems in rollup because of cyclic dependencies.

Reply:
@adamhock could you just import that directly as you did with other components in this file? Consistency is good

Reply:
Fixed